bobtemplates/plone/view.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""Generate view."""
+
+from bobtemplates.plone.base import base_prepare_renderer
+from bobtemplates.plone.base import git_commit
+from bobtemplates.plone.base import update_file
+from lxml import etree
+from mrbob.bobexceptions import SkipQuestion
+from mrbob.bobexceptions import ValidationError
+
+import os
+import stringcase
+
+
+def get_view_name(configurator, question):
+    """Generate view default name from python class"""
+    if configurator.variables['view_python_class']:
+        class_name = configurator.variables['view_python_class_name']
+        view_name = stringcase.snakecase(class_name).replace('_', '-')
+        question.default = view_name
+    else:
+        question.default = 'my-view'
+
+
+def get_template_name(configurator, question):
+    if configurator.variables['view_template']:
+        view_name = configurator.variables['view_name']
+        template_name = stringcase.snakecase(view_name)
+        question.default = template_name
+    else:
+        question.default = 'view'
+
+
+def check_python_class(configurator, question):
+    if not configurator.variables['view_python_class']:
```

Comment:
same here, dont't know what view_python_class means, is it the name of a class? that would be my frist guess.